fml/file.cc

```diff
@@ -0,0 +1,67 @@
+// Copyright 2018 The Flutter Authors. All rights reserved.
```

Comment:
Here and elswhere: keep as `The Chromium Authors` for now for consistency.

Reply:
Saying that would be factually incorrect though. We don't modify existing headers but use the latest message for new files.

Reply:
/cc @Hixie for clarification on all things licence-y since I'm not 100% sure what we're meant to do in these cases.